### ISSUE TITLE
Issue with .getPalette() on some images

### ISSUE THREAD

**Issue by AstroBoogie** [akfish/node-vibrant#48]
I get an error on some images when using .getPalette().

`TypeError: undefined is not an object (evaluating 'vbox.count') — mmcq.js:14`

The issue occurs in both Chrome and Safari, and it occurs randomly with different images. The images that cause this error are consistent. A lot of images that cause this issue have transparency, but this could be a coincidence. Here's the code I'm running:

```
    const logo = this.logo; // ref of img
    Vibrant.from(logo).getPalette((err, palette) => {
      console.log(palette, err);
    }
```

**Comment by prewk** [comment]
The following image causes `TypeError: Cannot read property 'count' of undefined` in Node: http://www.qgis.org/en/_static/images/users-docs.png

**Comment by prewk** [comment]
pull request #52

**Comment by akfish** [comment]
Fixed in v3.0
